vector/Dockerfile

```diff
@@ -34,6 +15,14 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN rpm --install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm  && \
     microdnf update && \
     microdnf --assumeyes install inotify-tools && \
-    microdnf clean all
-
-COPY --from=builder /build /
+    microdnf clean all && \
+    rpm --install "https://github.com/vectordotdev/vector/releases/download/v${PRODUCT}/vector-${PRODUCT}-1.$(arch).rpm" && \
```

Comment:
Can you mirror this in our repo and update the shell script we have to download a new Vector version?

Reply:
changed in bb3caf14f3e9075a3f42369f8f8048dd9ce85987